notebooks/explore_DL2.ipynb

```diff
@@ -15,6 +15,7 @@
     "from astropy.coordinates.erfa_astrom import ErfaAstromInterpolator, erfa_astrom\n",
```

Comment:
<p>Shall we suggest doing it from DL3 files in cases of very large datasets? We could set same cuts and energy ranges</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>